addons/website/static/src/js/editor/snippets.editor.js

```diff
@@ -170,5 +182,22 @@ weSnippetEditor.Class.include({
             tab: this.tabs.THEME,
         });
     },
+    /**
+     * Called when search input value changed > adapts the snippets grid.
+     */
+    _onSearchFilterInputChange: function () {
+        var self = this;
+        self._searchFilterValue = self.$('.o_snipet_search_filter_input').val().toLowerCase();
```

Comment:
- Instead of looking up the DOM for the input, you can directly accessing with `ev.target`, `ev` being the first parameter of your handler function.
- Do you really need to save the input value in `this`?